### ISSUE TITLE
SSE Feedback on AFF A900 Fan replacement

### ISSUE THREAD

**Issue by tullynet** [NetAppDocs/ontap-systems#136]
Page: [Swap out a fan - AFF A900](https://docs.netapp.com/us-en/ontap-systems/a900/fan_swap_out.html)

"You must replace the fan module within two minutes of removing it from the chassis. System airflow is disrupted and the controller module or modules shut down after two minutes to avoid overheating."

It appears that this has changed with this platform.

From the TOI-   
AFF A900 / FAS9500 will NOT automatically initiate shutdown process if the Fan Module is removed for 2min. or longer.  System will monitor internal temperature sensors and will initiate fan speed changes and shutdown process if critical thresholds have been reached or exceeded.

**Comment by tullynet** [comment]
https://docs.netapp.com/us-en/ontap-systems/a900/fan_swap_out.html

You must replace the fan module within two minutes of removing it from the chassis. System airflow is disrupted and the controller module or modules shut down after two minutes to avoid overheating.

Last updated 06/27/2022

**Comment by dougthomp** [comment]
You are correct, the current posted doc has the 2-minute warning text.  I have replaced it with:

It is a best practice to replace the power supply within two minutes of removing it from the chassis. The system continues to function, but ONTAP sends messages to the console about the degraded power supply until the power supply is replaced.  Yjr change will show up by Mondat 2/6.
